src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerOfflineDocumentVerified.java

```diff
@@ -237,15 +239,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
                                                                   CaseDetails<CaseData, State> beforeDetails) {
         log.info("{} midEvent callback invoked for Case Id: {}", CASEWORKER_OFFLINE_DOCUMENT_VERIFIED, details.getId());
         final CaseData data = details.getData();
-        if (RFI_RESPONSE.equals(data.getDocuments().getTypeOfDocumentAttached())
-            && (
-                data.getRequestForInformationList().getRequestsForInformation() == null
-                || data.getRequestForInformationList().getRequestsForInformation().isEmpty()
-            )
-        ) {
-            return AboutToStartOrSubmitResponse.<CaseData, State>builder()
-                .errors(Collections.singletonList(NO_REQUEST_FOR_INFORMATION_ERROR))
-                .build();
+        if (RFI_RESPONSE.equals(data.getDocuments().getTypeOfDocumentAttached())) {
+            if (data.getRequestForInformationList().getRequestsForInformation() == null
+                    || data.getRequestForInformationList().getRequestsForInformation().isEmpty()
+            ) {
+                return AboutToStartOrSubmitResponse.<CaseData, State>builder()
+                    .errors(Collections.singletonList(NO_REQUEST_FOR_INFORMATION_ERROR))
```

Comment:
Do we need to include `.data()` here or is this an optional part just like .state?

Reply:
We typically don't include data on an error return.